data/map.txt

```diff
@@ -23587,6 +23587,8 @@ system "Ultima Thule"
 	trade Medical 732
 	trade Metal 417
 	trade Plastic 360
+	fleet "Small Human Merchants (Hai)" 15000
+	fleet "Large Human Merchants (Hai)" 20000
```

Comment:
I'd keep Ultima Thule as an empty system. Therefore if the player runs out of fuel in Ultima Thule, there only option is to head through the wormhole. (Helps to make Hai space a little more discoverable.)

Reply:
Okay. Please just make sure you fix refueling at Waypoint before 0.9.12. The fact that it takes only 400 fuel to get safely to Hai space is pretty important.

Reply:
> Please just make sure you fix refueling at Waypoint before 0.9.12. 

Well isn't that what this PR is for? So you should, you know, address the review? >insert James emote<

Reply:
Somehow I didn't notice there was a review.

Reply:
All good. I got it.